rules/framework/vfs_overlay.bzl

```diff
@@ -43,6 +43,20 @@ def _make_root(ctx, framework_name, root_dir, extra_search_paths, module_map, hd
             "external-contents": module_map[0].path,
         })
 
+    if len(swiftmodule):
```

Comment:
Does this code do anything if it's removed? I wasn't 100% sure if it looked in this directory for swiftmodules? It likely will conflict with the top level dir for swiftmodule

Reply:
I think the test `if len(swiftmodule) > 0:` is the same conditional test as `if len(swiftmodule)`. 
Mainly, the existing code puts them at the top level, so I was wondering if there was a difference in nesting them under the framework. 

Here is the change I'm referring to:  https://github.com/bazel-ios/rules_ios/pull/280/files#diff-450b1330485549712d2fd4e65b2b3fd0a7692fe8f4ec43a0b7b60bbb739313afR52